openedx/features/badging/tests/test_views.py

```diff
@@ -82,11 +80,16 @@ def test_my_badges_denies_anonymous(self):
         response = Client().get(path=path)
         self.assertRedirects(response, '{}?next={}'.format(reverse('signin_user'), path))
 
+        path = reverse('trophycase')
+        response = Client().get(path=path)
+        self.assertRedirects(response, '{}?next={}'.format(reverse('signin_user'), path))
+
     def test_my_badges_invalid_course_id(self):
         """
         Test my badges with invalid course id. Assert that an error is raised
         :return: None
         """
+        self.client.login(username=self.user.username, password='test')
```

Comment:
I see at line number `96-97` we are still using `request_factory`, can't we use client here?

Reply:
Corrected it now.